<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCountry</class>
 <widget class="QDialog" name="DialogCountry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnNewCountry">
         <property name="text">
          <string>New country</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRemoveCountry">
         <property name="text">
          <string>Remove country</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnReload">
         <property name="text">
          <string>Reload</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnClose">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <action name="action_add_new_country">
   <property name="text">
    <string>Add new country</string>
   </property>
  </action>
  <action name="action_remove_selected_country">
   <property name="text">
    <string>Remove selected country</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
